app/0.14.x/getting-started/quickstart.md

```diff
@@ -69,7 +69,7 @@ Issue the following command to [reload][CLI] Kong without downtime:
 $ kong reload
 ```
 
-## Next Steps
+## Next steps
```

Comment:
This capitalization inconsistency also occurs in prior docs versions - please correct them all.